clang/test/CIR/CodeGen/complex.c

```diff
@@ -0,0 +1,243 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+// RUN: %clang_cc1 -x c++ -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
```

Comment:
it would be nice to have a `test/CIR/IR/complex.cir` test for printing and parsing as well.